src/main/kotlin/com/sourcegraph/cody/history/HistoryTree.kt

```diff
@@ -68,10 +60,21 @@ class HistoryTree(
             AllIcons.Actions.GC,
             isEnabled = { true },
             ::removeAllLeafs))
-
-    PopupHandler.installPopupMenu(tree, group, "ChatActionsPopup")
-    EditSourceOnDoubleClickHandler.install(tree, ::selectLeaf)
-    setContent(ScrollPaneFactory.createScrollPane(tree))
+    ApplicationManager.getApplication().invokeLater {
```

Comment:
Can you put previous init code in invokeLater as well? :)

Reply:
Sure! However I don't think it's necessary. I assume it will be safer that way.

Reply:
Hmm, it just seems to me that there is no need to include model construction or popup initialization here. Both of these things are built in isolation (from UI perspective) and effectively used within the invokeLater block. Do we want to add this **for certainty** or for a specific reason?

I have no objections to doing it, but it's possible that I don't understand something.